src/main/java/summarizer/team/TeamSummarizerFactory.java

```diff
@@ -0,0 +1,17 @@
+package summarizer.team;
+
+import team.ITeamMember;
+import equipments.AbstractEquipment;
+
+import java.util.List;
+import java.util.Map;
+
+public class TeamSummarizerFactory {
+    public String getTeamSummary(TeamSummarizerType teamSummarizerType, Map<ITeamMember, List<AbstractEquipment>> teamEquipments){
```

Comment:
SRP- what is this class's responsibility? is it responsible for summarizing or just object creation?